### ISSUE TITLE
Failed to compile in Windows 10

### ISSUE THREAD

**Issue by stephenong** [stm32duino/arm-none-eabi-gcc#1]
**Describe the bug**
Error in compiling

**To Reproduce**
The Blink sample app

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Arduino IDE version: 1.8.9
 - STM32 core version: 1.6.0

**Board (please complete the following information):**
 - Name: BluePill

**Additional context**
There is a white space in my windows windows path

Error message:

```
c:/users/stephen ong/appdata/local/arduino15/packages/stm32/tools/arm-none-eabi-gcc/8-2018-q4-major/bin/../lib/gcc/arm-none-eabi/8.2.1/../../../../arm-none-eabi/bin/ld.exe: cannot find ong/appdata/local/arduino15/packages/stm32/tools/arm-none-eabi-gcc/8-2018-q4-major/bin/../lib/gcc/arm-none-eabi/8.2.1/lto-wrapper.exe: No such file or directory
```

**Comment by stephenong** [comment]
The workaround of installing portable Arduino IDE works. Thanks!

**Comment by fpistm** [comment]
This is now fix with new release 8.2.1-1.7:
https://github.com/stm32duino/arm-none-eabi-gcc/releases/tag/8.2.1-1.7

Available thanks 1.6.1 Core release:
https://github.com/stm32duino/Arduino_Core_STM32/releases/tag/1.6.1
